tasks/38_max-product/max-product.js

```diff
@@ -0,0 +1,9 @@
+export function maxProduct(arr) {
+  let max = arr[0]*arr[1];
+  for (let i=1; i<arr.length - 1; i++) {
```

Comment:
Why not `for (let i = 0; i < arr.length; i++) {` ?

Reply:
I start with `let i = 1` because I take the multiplication of the first two elements as the default `max` value outside of the `for` loop, and I break the loop at `i < art.length - 1` because there is no need for the extra iteration.